src/customize-pop/gnome-tweaks-extensions/user-themes.md

```diff
@@ -0,0 +1,94 @@
+# Managing User Themes
+
+User themes include changes to window animations, desktop color schemes, window designs, icons, and workspaces. You can install custom GNOME Themes from opendesktop.org. The Pop!\_OS default themes can be re-applied using GNOME Tweaks or the by using Terminal commands.
+
+>**Note**: Applying custom GNOME theming options may cause system instability or glitchy behavior. Additionally, custom themes are not subject to testing by the System76 Quality Assurance team or application developers.
+
+## Enabling User Themes
+
+1. Launch the GNOME Extension Manager.
+
+    ![Launch GNOME Extension Manager](/images/user-themes/launch-ext-manager.png)
+
+2. Click `Browse` and search for `user themes`, then click `Install`.
```

Comment:
Searching for the ID `user-theme@gnome-shell-extensions.gcampax.github.com` instead of the name will keep similar options (e.g. `User Themes X`) from showing up.

Reply:
Updated with this search term and matching screenshot.